### PR TITLE
Updated terminology - 'bond(ing) slaves' into 'bond ports'

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Description

...as inspired by the YaST GUI term

### PR creator: Are there any relevant issues/feature requests?

* jsc#SLE-22017

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/net_bonding.xml [hunk 1]
```diff
@@ -56,6 +56,7 @@
     <para>
      Traffic is split between all available interfaces based on the following
      policy: <literal>[(source MAC address XOR'd with destination MAC address
+      <!-- 2022-04-01 tbazant: how to update this one? -->
```
Comment (tahliar) [inline xml/net_bonding.xml:59]: I'm not really sure myself - I didn't change it in the Smart Doc either :confused:

**Comment by tahliar** [comment]
I don't really like "bond ports" as a term, but that's out of our hands and is better than slave, at least. So this looks fine to me.

**Review by tahliar — APPROVED** [review]

File: xml/net_bonding.xml (hunk 1 above)
Comment (mchf) [reply xml/net_bonding.xml:59]: in some comments in the code i replaced something similar by "number of devices included in the bonding" ... Far from perfect, too long ... but only i could imagine.

File: xml/net_bonding.xml [hunk 2]
```diff
@@ -229,12 +230,13 @@
           STARTMODE='auto' # or 'onboot'
           BOOTPROTO='static'
           IPADDR='192.168.0.1/24'
+          <!-- 2022-04-01 tbazant: update config options as well? how? -->
```
Comment (mchf) [inline xml/net_bonding.xml:233]: as far as i know wicked has not introduced inclusive naming compatible replacement yet.

File: xml/net_bonding.xml [hunk 3]
```diff
@@ -254,11 +256,11 @@ ifcfg-eth1
   <para>
    <literal>BOOTPROTO=none</literal> uses the <command>ethtool</command>
    options (when provided), but does not set the link up on <command>ifup
-   eth0</command>. The reason is that the slave interface is controlled by the
-   bond master.
+   eth0</command>. The reason is that the bond port interface is controlled by the
+   bond parent.
```
Comment (mchf) [inline xml/net_bonding.xml:260]: I personally don't like ```parent``` replacement for the master device. I've also used that as last instance only. So, I'd most probably use ```bond device``` here.

File: xml/net_dns.xml [hunk 4]
```diff
@@ -856,12 +856,12 @@ too easy to do. -->
 };
 
 zone "localhost" in {
-       type master;
+       type primary;
```
Comment (mchf) [inline xml/net_dns.xml:859]: It is snippet of a configuration file for the dns server. So, this is wrong as far as the dns service has not implemented new keyword (i don't know). 

However, talking about yast, yast exports old ```master``` keyword into the config.

File: xml/net_dns.xml [hunk 5]
```diff
@@ -856,12 +856,12 @@ too easy to do. -->
...
        file "localhost.zone";
 };
 
 zone "0.0.127.in-addr.arpa" in {
-        type master;
+        type primary;
```
Comment (mchf) [inline xml/net_dns.xml:864]: dtto

File: xml/net_dns.xml [hunk 6]
```diff
@@ -1039,7 +1039,7 @@ zone "." in {
    <example xml:id="ex-meine-domain">
     <title>Zone entry for example.com</title>
 <screen>zone "example.com" in {
-      type master;
+      type primary;
```
Comment (mchf) [inline xml/net_dns.xml:1042]: dtto

File: xml/net_bonding.xml (hunk 2 above)
Comment (tbazant) [inline xml/net_bonding.xml:233]:
```suggestion
```

File: xml/net_bonding.xml (hunk 3 above)
Comment (tbazant) [inline xml/net_bonding.xml:260]:
```suggestion
   bond device.
```

File: xml/net_dns.xml (hunk 4 above)
Comment (tbazant) [inline xml/net_dns.xml:859]:
```suggestion
       type master;
```

File: xml/net_dns.xml (hunk 5 above)
Comment (tbazant) [inline xml/net_dns.xml:864]:
```suggestion
        type master;
```

File: xml/net_dns.xml (hunk 6 above)
Comment (tbazant) [inline xml/net_dns.xml:1042]:
```suggestion
      type master;
```

**Review by tbazant — COMMENTED** [review]
Included suggested changes.

File: xml/net_dns.xml [hunk 7]
```diff
@@ -1069,21 +1070,21 @@ zone "." in {
    </para>
    <variablelist>
     <varlistentry>
-     <term>type master;</term>
+     <term>type primary;</term>
      <listitem>
       <para>
-       By specifying <literal>master</literal>, tell BIND that the zone is
+       By specifying <literal>iprimary</literal>, tell BIND that the zone is
```
Comment (dariavladykina) [inline xml/net_dns.xml:1076]:
```suggestion
       By specifying <literal>primary</literal>, tell BIND that the zone is
```

File: xml/net_dns.xml [hunk 8]
```diff
@@ -44,28 +44,28 @@
...
        <listitem>
         <para>
-         A slave zone is a copy of the master zone. The slave zone DNS server
-         obtains its zone data with zone transfer operations from its master
-         server. The slave zone DNS server responds authoritatively for the
-         zone as long as it has valid (not expired) zone data. If the slave
+         A secondary zone is a copy of the primary zone. The secondary zone DNS server
+         obtains its zone data with zone transfer operations from its primary
+         server. The secondary zone DNS server responds authoritatively for the
+         zone as long as it has valid (not expired) zone data. If the secondary server
```
Comment (dariavladykina) [inline xml/net_dns.xml:68]:
```suggestion
         zone as long as it has valid (not expired) zone data. If the secondary server
```
```suggestion
         zone if it has valid (not expired) zone data. If the secondary server
```

File: xml/net_dns.xml [hunk 9]
```diff
@@ -498,32 +498,32 @@
...
    </sect3>
-   <sect3 xml:id="sec-dns-yast-expert-edit-master">
-    <title>DNS zones (editing a master zone)</title>
+   <sect3 xml:id="sec-dns-yast-expert-edit-primary">
+    <title>DNS zones (editing a primary zone)</title>
     <para>
-     To edit a master zone, select <guimenu>DNS Zones</guimenu>, select the
-     master zone from the table, and click <guimenu>Edit</guimenu>. The dialog
+     To edit a primary zone, select <guimenu>DNS Zones</guimenu>, select the
+     primary zone from the table, and click <guimenu>Edit</guimenu>. The dialog
```
Comment (dariavladykina) [inline xml/net_dns.xml:518]:
```suggestion
     primary zone from the table and click <guimenu>Edit</guimenu>. The dialog
```

File: xml/net_dns.xml [hunk 10]
```diff
@@ -1279,7 +1280,7 @@ dns6         IN A6  0    &dnsipv6;</screen>
     <para>
      The <systemitem>serial number</systemitem> is a 10 digit number. It must
```
Comment (dariavladykina) [inline xml/net_dns.xml:1281]:
```suggestion
     The <systemitem>serial number</systemitem> is a 10-digit number. It must
```

File: xml/net_dns.xml [hunk 11]
```diff
@@ -1279,7 +1280,7 @@ dns6         IN A6  0    &dnsipv6;</screen>
...
      be changed each time this file is changed. It is needed to inform the
-     secondary name servers (slave servers) of changes. For this, a 10 digit
+     secondary name servers (secondary servers) of changes. For this, a 10 digit
```
Comment (dariavladykina) [inline xml/net_dns.xml:1283]:
```suggestion
     secondary name servers (secondary servers) of changes. For this, a 10-digit
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some corrections.

**Pull request merged**